docs/contribution/contribution.md

```diff
@@ -0,0 +1,140 @@
+   need you to sign a Contributor License Agreement (CLA). Code doesn't just
+   mean "components"; if you're contributing to our website docs, you're
+   contributing code. ;)
+3. **Development environment:** If you haven't already, fork and clone whichever
+   repo you want to contribute to. Then, create a new branch and add your
+   contribution in it. Checkout our
+   [Developer Handbook](../developer-handbook.md) to read up on our best coding
+   practices and proper commit messages.
+4. **Pull request:** Submit a PR. Be sure to fill out the template properly.
+5. **Approval:** Get PR approved by design and developers, or make any necessary
+   changes for approval. This process may be quick or take a few iterations of
+   feedback-update.
+6. **Documentation:** After design and dev have approved and merged PR, update
+   any website documentation if necessary. One of the best examples for this is
+   if you're contributing to component work which has website documentation
+   related to your contribution.
+
+Here are some contribution quick tips:
+
+- **Do** check repos for existing issues.
+- **Do** fill out the required template for contributions entirely; this
+  pertains to both issues and PRs.
+- **Do** add or update tests for any contributions that require it.
+- **Do** follow existing coding and writing styles.
+- **Do** follow proper commit messages syntax.
+- **Do not** branch off another branch.
+- **Do not** include unrelated changes in the same PR.
+- **Do not** create one massive PR if it can be broken up into smaller PRs.
+
+### What projects can I contribute to?
+
+- [`carbon-components`](https://github.com/carbon-design-system/carbon/tree/master/packages/components)
+- [`carbon-components-react`](https://github.com/carbon-design-system/carbon/tree/master/packages/react)
+- [`carbon-website`](https://github.com/carbon-design-system/carbon-website)
+- [`gatsby-theme-carbon`](https://github.com/carbon-design-system/gatsby-theme-carbon)
+- [`carbon-design-kit`](https://github.com/carbon-design-system/carbon-design-kit)
+- [`@carbon/elements`](https://github.com/carbon-design-system/carbon/tree/master/packages/elements)
+- [`@carbon/type`](https://github.com/carbon-design-system/carbon/tree/master/packages/type)
+- [`@carbon/colors`](https://github.com/carbon-design-system/carbon/tree/master/packages/colors)
+- [`@carbon/grid`](https://github.com/carbon-design-system/carbon/tree/master/packages/grid)
+- [`@carbon/icons`](https://github.com/carbon-design-system/carbon/tree/master/packages/icons)
+- [`@carbon/layout`](https://github.com/carbon-design-system/carbon/tree/master/packages/layout)
+- [`@carbon/motion`](https://github.com/carbon-design-system/carbon/tree/master/packages/motion)
+- [`@carbon/themes`](https://github.com/carbon-design-system/carbon/tree/master/packages/themes)
+- [`@carbon/icons-react`](https://github.com/carbon-design-system/carbon/tree/master/packages/icons-react)
```

Comment:
isn't there other `@carbon/icons-*` packages as well? (WH specifically uses the handlebars one)